scripts/gan/stylegan/README.md

```diff
@@ -2,7 +2,24 @@
 
 **Train StyleGAN**
 
-Instructions coming soon.
+First, prepare the dataset for training. Download the FFHQ dataset from (https://github.com/NVlabs/ffhq-dataset) and save it to DATASET_PATH.
+LMDB_PATH: directory to save the output dataset, N_WORKER: number of workers, DATASET_PATH: the downloaed FFHQ images folder path 
+```bash
+python prepare_data.py --out LMDB_PATH --n_worker N_WORKER DATASET_PATH
+```
+
+Second, train the StyleGAN for FFHQ dataset.
+```bash
+python train.py --path LMDB_PATH --sched
+```
+
+*Notes for training*
+1) The original tensorflow implementation can't be 100% converted by MXNet. Two functions are missing, gradient penalty and blur. The lack of gradient penalty can cause mode collapse while training, so it is neccessary to tune the learning rate based on the number of GPUs and apply early stop. The lack of blur function results in the low image quality and this is one of the important reasons that high-resolution images can't be generted via our implementation. 
+2) This is an unstable version of StyleGAN. We've tested the training by using 8 GPUs and single GPU. Single GPU can be problematic. The following images are generated by a model trained with 8 GPUs. 
```

Comment:
This is an unstable version of StyleGAN -> The training of StyleGAN is not stable at this moment due to the aforementioned reasons.